samples/subsys/display/lvgl/src/main.c

```diff
@@ -47,7 +47,12 @@ void main(void)
 	lv_obj_align(count_label, NULL, LV_ALIGN_IN_BOTTOM_MID, 0, 0);
 
 	lv_task_handler();
-	display_blanking_off(display_dev);
+
+	if (IS_ENABLED(CONFIG_EK79652)) {
+		display_refresh(display_dev, PARTIAL_REFRESH);
```

Comment:
This is basically still a device-specific ifdef inside what is supposed to be a generic sample. Such things are generally bad practice in zephyr. Either the device API for displays should be able to tell the user if the display_refresh() or display_blanking_off() API should be used for a particular device, or there should be some common init function here that does the right thing. Doing this in a device-specific way won't scale.

Reply:
yes, I am completely agree that not to use device-specific ifdef in generic sample. this can be rectified only if the new API chnages are reviewed, since this particular display is having different registers for display refresh(partial or full), we need user input to do the display refresh accordingly. this API is mainly introduced to avoid complete display refresh when only part of the display needs refresh, say if we are using number counter with static text message , we don't need entire display to refresh when only the part where number counter is displayed. @jfischer-no need inputs from your side.

Reply:
> @jfischer-no need inputs from your side.

ping

Reply:
Again, NACK to API changes, strong NACK to this change in the sample.